plugins/linux/nsplugin/ns_handler.go

```diff
@@ -69,7 +74,8 @@ type NsHandler struct {
 }
 
 // Init namespace handler caches and create config namespace
-func (plugin *NsHandler) Init(logger logging.PluginLogger, msChan chan *MicroserviceCtx, ifNotif chan *MicroserviceEvent) error {
+func (plugin *NsHandler) Init(logger logging.PluginLogger, ifHandler linuxcalls.NetlinkAPI, sysHandler SystemAPI,
```

Comment:
Now that we have so many handlers we should rename `NsHandler` to `NamespaceConfigurator` to keep consistency with other configurators and not confuse with these new handlers.

Reply:
Actually it seems to me that it's not really `NamespaceConfigurator` since it's more like linuxcalls for namespace handling. We will need some discussion how to split this package because it seems this `nsplugin` package now serves several different purposed and doesn't seem consistent with other configurators.

Reply:
Yes, this is not a plugin/configurator in the true sense, it's more like generic handler for namespaces and microservices which provides methods for every other plugin.